scripts/template_upload_rhevm.py

```diff
@@ -145,7 +145,7 @@ def template_from_ova(api, username, password, rhevip, edomain, ovaname, ssh_cli
         print("RHEVM:{} Warning: found another template with this name.".format(provider))
         print("RHEVM:{} Skipping this step. Attempting to continue...".format(provider))
         return
-    version_cmd = 'rpm -qa| grep rhevm-image-uploader'
```

Comment:
@psav: nope, "image-uploader " rpm name changed for 4.0. but this string would be enough to grep rpm on other versions.